prisma/migrations/01_functions/migration.sql

```diff
@@ -1,6 +1,3 @@
-SET
-  GLOBAL log_bin_trust_function_creators = 1;
```

Comment:
Need root to run this, but I think this is safe to skip.

As far as I understand this is mostly used for replication (which we don't have I think?), and can be required for some specific functions, but all current functions can be created successfully without it so should be fine.

If my understanding is correct, we could turn it of manually for :norway: prod for perfect alignment.

Reply:
I'm unable to verify this issue now while testing though, so I guess this is good!

Reply:
It wasn't too tricky to debug, so hopefully if someone encounters it we can add some documentation about it.